packages/lib/RotatingLogs.ts

```diff
@@ -0,0 +1,44 @@
+import shim from './shim';
+import { Stat } from './fs-driver-base';
+
+export default class RotatingLogs {
+
+	private logFilesDir = '';
+	private nonActiveLogFileMaximumDaysAge = 90 * 24 * 60 * 60 * 1000;
+	private activeLogFileMaximumSizeInBytes: number = 1024 * 1024 * 100;
```

Comment:
Rename to `maxFileSize`. It's good to give descriptive variable name but in this case it's distracting